src/pages/components/leaving-ibm.mdx

```diff
@@ -20,7 +20,15 @@ import ComponentFeedback from "components/ComponentFeedback";
 
 ## Default
 
-When a use clicks on an external link it will launch the Leaving IBM overlay to help users understand they are leaving IBM.com. It includes a brief message about a possible change in terms, and the host name of the external site is displayed to be transparent.
+Do not apply this notice to all external links. Use this notice only when the user leaves the ibm.com domain to complete a service or transaction (such as commerce or to register for a conference or webinar) on behalf of IBM.  For example, when purchasing a product from IBM Marketplace, a user may choose to use PayPal as a payment method. When they leave ibm.com to go to the PayPal site to complete the transaction, the Leaving IBM overlay would be displayed.
+
+If the destination URL contains parameters, it must be URL encoded to work properly.
+
+The link URL must be preceded with "//www.ibm.com/links/?url="
```

Comment:
This sentence is showing as a link on the page. Maybe wrap in a code snippet? 
` //www.ibm.com/links/?url=` 

![image](https://user-images.githubusercontent.com/15144993/115594909-4ae83400-a2a4-11eb-863a-54803e13057c.png)

Reply:
@shixiedesign can you make it into code snippet for me?

Reply:
@ljcarot Code-snippet update done.